test/hga/HGA_Regression.ipynb

```diff
@@ -720,129 +719,6 @@
     "    print('Skipping test: HGA regression tests not configured for this environment.')"
    ]
   },
-  {
-   "cell_type": "markdown",
-   "id": "8167880c",
-   "metadata": {},
-   "source": [
-    "### AVNIR test case 10:\n",
-    "\n",
-    "This test combines a bounding box spatial subset, across the Equator, with a variable subset for a single variable. The output should be in a `.png` format.\n",
-    "\n",
-    "The input `.zip` file contains 4 Image GeoTIFFs, one per band each representing a spectrum range. \n",
-    "\n",
-    "For a `.png` output, the authority and projection can't be derived from the output file, so several of the tested properties are `None` for this test case.\n",
-    "\n",
-    "## Test Disabled\n",
-    "\n",
-    "TODO [06/21/2022]: We noticed some strange behavior with creating png output\n",
```

Comment:
Well, it's been over two years since this test was commented out - GIF requests have been problematic in that time and no one has been raising that to us. Pretty indicative of usage.